### PR TITLE
fixed documentation for face-detection-retail-0004

### PR DESCRIPTION
**Pull request by atsyplen**

### CODE REVIEW

File: intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ head from 1/16 scale feature map has nine clustered prior boxes.
 
 | Metric                                                        | Value                   |
 |---------------------------------------------------------------|-------------------------|
-| AP ([WIDER](http://shuoyang1213.me/WIDERFACE/)                | 83.00%                  |
+| AP ([WIDER])(http://shuoyang1213.me/WIDERFACE/)               | 83.00%                  |
```
Comment (zmaslova) [inline intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md]: `AP [WIDER](http://shuoyang1213.me/WIDERFACE/)` - this is correct change

Comment (zmaslova) [reply intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md]: or `([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/))`

Comment (kchechil) [reply intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md]: This one should be the correct one
([WIDER](http://mmlab.ie.cuhk.edu.hk/projects/WIDERFace/))

**Comment by kchechil** [comment]
Jenkins please retry a build

File: intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md (hunk 1 above)
Comment (atsyplen) [reply intel_models/face-detection-retail-0004/description/face-detection-retail-0004.md]: fixed

**Comment by atsyplen** [comment]
Jenkins please retry a build

**Pull request merged**